public/raw_articles/tba-bulk-data.md

```diff
@@ -0,0 +1,185 @@
+[//]: <author> (Andrew Schreiber)
+[//]: <title> (TBA API For Beginners)
+[//]: <updated> (10/01/2020)
+
+[[toc]]
+
+## Accessing Bulk TBA Data
+
+As you may have noticed, accessing the TBA API is pretty easy if you need to see one team or one event but what if you wanted to see something else, like how often Team 254 has Red Bumpers. For stuff like that TBA has an mirror of their data to a publicly queryable data set available at https://thebluealliance.com/bigquery.
```

Comment:
For whatever reason the [MD->html parser](https://github.com/markdown-it/markdown-it) isn't properly converting bare URLs right now, so could you just convert this to [link to bigquery](link to bigquery)?